docs/source/topics/ref_dns-configuration.adoc

```diff
@@ -29,13 +29,29 @@ This configuration is used on Fedora 33 or newer, and on Ubuntu Desktop editions
 * `systemd-resolved` configuration is done through a NetworkManager dispatcher script in `/etc/NetworkManager/dispatcher.d/99-crc.sh`:
 +
 ----
+#!/bin/sh
 resolvectl domain crc ~testing
 resolvectl dns crc 192.168.130.11
 resolvectl default-route crc false
 
 exit 0
 ----
 
+[NOTE]
+====
+:rhel-resolved-docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/using-different-dns-servers-for-different-domains_configuring-and-managing-networking
```

Comment:
Would you mind placing this attribute in the attributes file instead of here? The file is in `docs/source/meta/product-attributes.adoc`. Thank you!

Everything else LGTM. :+1:

Reply:
Attribute moved to `product-attributes.adoc`.